OMPython/__init__.py

```diff
@@ -238,8 +239,11 @@ def sendExpression(self, command):
             self._omc = None
             return result
           else:
-            answer = OMTypedParser.parseString(result)
-            return answer
+            try:
+              answer = OMTypedParser.parseString(result)
+              return answer
+            except:
```

Comment:
1. It's typically bad python style to catch all exceptions without qualification, you should probably limit this to PyPArsingExceptions or whatever the type of the exceptions you expect are.
2. In the case of the exception, shouldn't you return some info about the exception, e.g. its message? You just return the string you wanted to parse in the first place.